examples/index.json

```diff
@@ -1433,6 +1433,13 @@
         "verifyOutput": true,
         "isLearnByExample": true
       },
+      {
+        "name": "Named worker with on fail",
```

Comment:
Let's fix the order as per https://github.com/ballerina-platform/ballerina-dev-website/issues/8856#issuecomment-2044360385